src/components/DeleteConfirm.js

```diff
@@ -0,0 +1,21 @@
+import Popup from "./Popup.js";
+
+export default class DeleteConfirm extends Popup {
+  constructor(popupSelector, deleteCardForm, handleCardDelete) {
+    super({ popupSelector });
+    this._handleCardDelete = handleCardDelete;
+    this._deleteCardForm = deleteCardForm;
+  }
+
+  submitHandler(handler) {
```

Comment:
Function names should start with a **verb** in infinitive (there is such a point in the checklist). It’s common in coding to do it so that functions could `speak` for themselves about what they do.


`set...`